udacitydl/udacitydl.py

```diff
@@ -133,10 +97,27 @@ def download_course(self, cname, dest_dir="."):
             for fname, tfname in download_dict.iteritems():
                 try:
                     print "    * Downloading ", fname, "..."
-                    self.download(tfname, target_dir=resource_dir, target_fname=fname)
+                    download_file(tfname, resource_dir, fname)
                 except Exception as e:
                     print "     - failed ", fname, e
 
+def download_file(url, path ,fn):
```

Comment:
Yes, I have downloaded some courses using it.
The 403 could be because urllib is blacklisted.